src/julia-parser.scm

```diff
@@ -159,7 +159,7 @@
 (define reserved-word? (Set reserved-words))
 
 (define closing-token?
-  (let ((closer? (Set '(else elseif catch finally #\, #\) #\] #\} #\;))))
+  (let ((closer? (Set '(else elseif catch finally #\, #\) #\] #\} #\⟩ #\⌋ #\⌉ #\;))))
```

Comment:
Doesn't this construct a `Set` object every time `closing-token?` is called?   It seems like for performance we may want to define some global sets for this function.

Reply:
If you look closely, the let is outside the lambda.